debian/securedrop-client.lintian-overrides

```diff
@@ -0,0 +1,22 @@
+securedrop-client: arch-independent-package-contains-binary-or-object
+# This is intentional
```

Comment:
Nit: This file takes a moment to scan, because it looks like it's—

```
directive
# comment
```

—but it's actually—

```
# comment
directive
```

—after the first directive, which isn't commented.  Maybe add an extra newline before each directive?

```directive

# comment
directive

# comment
directive

# ...
```

Reply:
Done!